apps/backend/alembic/versions/66cc68f4e112_fix_configuration_snapshots_id_column_.py

```
@@ -0,0 +1,67 @@
+# revision identifiers, used by Alembic.
+revision: str = '66cc68f4e112'
+down_revision: Union[str, Sequence[str], None] = 'af4ba97c1822'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    """Upgrade schema - fix configuration_snapshots ID column to UUID."""
+    # Drop the existing table since it's empty and we need to recreate with UUID
+    op.drop_table('configuration_snapshots')
+    
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)